neural_network_block.py

```diff
@@ -91,7 +92,7 @@ class NeuralNetwork(Block):
     network_config = ObjectProperty(NetworkConfig,
                                     title='ANN Configuration',
                                     defaul=NetworkConfig())
-    version = VersionProperty('0.2.0')
+    version = VersionProperty('0.3.0')
```

Comment:
since the version property changed, also update the `spec.json` and `release.json` by doing `nio buildspec tensor_flow` and `nio buildrelease tensor_flow` from the project directory